tests/e2e/test_testing.py

```diff
@@ -44,3 +45,24 @@ def test_expect_revert(protostar, copy_fixture):
     )
     assert "[error_type] RANDOM_ERROR_NAME" in result
     assert "Unknown location" not in result
+
+
+def test_loading_cairo_path_from_config_file(protostar, my_private_libs_setup):
+    (my_private_libs_dir,) = my_private_libs_setup
+
+    # TODO: Protostar shouldn't crash when imports are incorrect
+    # https://github.com/software-mansion/protostar/issues/230
```

Comment:
Please remove this TODO, we such keep such stuff in the issue tracker. You can add permalink there